pytorch_lightning/trainer/trainer.py

```diff
@@ -295,7 +295,8 @@ def __init__(
         super().__init__()
         self._device_type = DeviceType.CPU
         self._distrib_type = None
-        self._running_stage = None
+        self._running_stage = RunningStage.UNDEFINED
+        self.is_predicting = False
```

Comment:
No we can't in the current state of the codebase. We are using self.trainertesting for routing the code, and if we set self._running_stage == RuningStage.PREDICTING, it would route to validation.

I plan to properly refactor routing in a follow up PR as it is currently a big mess.